signer/core/apiv2.go

```diff
@@ -0,0 +1,259 @@
+package core
```

Comment:
Not sure if `apiv2.go` is a good name -- perhaps signed_data.go or something more descriptive

Reply:
Oh no so that's just temporary until the first version is done, it's just easier to have everything there. I was thinking to move it over to `api.go`, but indeed it may get clunky. `signed_data` sounds good.